apps/labs/pages/blog/[slug].tsx

```diff
@@ -1,6 +1,3 @@
-import { readdir } from 'fs/promises';
-import path from 'path';
-
```

Comment:
Unused imports.

Reply:
I think it should, but I don't have time right now to stop everything and investigate this. I have a suspicion that there were a few things that got into (or out of) the codebase that shouldn't have because of all of the activity on the repo in the past week and consequent merge conflicts being resolved incorrectly.